config/processCcd.py

```diff
@@ -18,8 +18,40 @@
 #This is just an example of how to change a parameter.
 #You don't have to write out the ISR image (but it
 #can be useful to do so).
-config.isr.doWrite = True
-config.isr.doBias = True
+
+# Apply bias frame correction?
+config.isr.doBias=True
+
+# Apply dark frame correction?
 config.isr.doDark=True
-config.isr.doFlat=True
+
+# Apply flat field correction?
+config.isr.doFlat=False
+
+# Apply fringe correction?
+config.isr.doFringe=False
+
+# Apply correction for CCD defects, e.g. hot pixels?
+config.isr.doDefect=False
+
+# Apply a distortion model based on camera geometry to the WCS?
+config.isr.doAddDistortionModel=True
```

Comment:
This refers to applying a distortion model that is defined in camera.py, rather than the calculation of SIP terms. I don't think you we have a distortion model at the moment, so I suspect that this should be set to `False`.